src/tests/system/tests/test_sss_ssh_knownhosts.py

```diff
@@ -1,7 +1,7 @@
 """
 sss_ssh_knownhosts tests.
 
-:requirement: None
+:requirement: Tools
```

Comment:
```suggestion
:requirement: Support 'KnownHostsCommand' and deprecate 'sss_ssh_knownhostsproxy'
```
This should link to: https://issues.redhat.com/browse/RHEL-17913 for traceability.

The import for this did not seem to happen yet as the jira does not have fields populated properly.